ci-operator/step-registry/ipi/install/workers-rhel7/prep/ipi-install-workers-rhel7-prep-commands.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
```

Comment:
`prep` is not a very clear name.. is there something better that can define this. If this is multiple steps lumped together we could break them up too to be specific ones.

Reply:
Prep does two things to prepare the host,
- Copy the ops-mirror.pem to /var/lib/yum
- Create the rhel-7-server and ose repo files in /etc/yum.repos.d

There is only one playbook run to do this so that is why it has it's own step.  Based on the above, what do you suggest is a better name?  `repos`?

Reply:
Also, things the customer need to do to 'prepare the host for OCP' would be done here.  Right now that is only setting up repos.  

https://docs.openshift.com/container-platform/4.5/machine_management/adding-rhel-compute.html#rhel-preparing-node_adding-rhel-compute

Reply:
Maybe `prep-yum-repos` ??

Reply:
I reorganized the steps a bit more and moved the remaining worker-rhel7 steps out of `ipi` and `ipi-install` since it was confusing to be under `ipi` and `install`.  I renamed the `prep` step to `repos` to hopefully make that more clear.  This is a summary of the new steps and organization.  

```
ipi
  aws
    pre
      workers-rhel7
        -chain.yaml

openshift
  e2e
    aws
      workers-rhel7
        -workflow.yaml

ssh-bastion
  -ref.yaml

workers-rhel7
  aws-provision
    -ref.yaml
  repos
    -ref.yaml
  scaleup
    -ref.yaml
  -chain.yaml

```